pollen_vision/pollen_vision/camera_wrappers/depthai/cam_config.py

```diff
@@ -109,31 +109,39 @@ def get_calib(self) -> dai.CalibrationHandler:
         """Returns a dai.CalibrationHandler object with all the camera's calibration data."""
         return self.calib
 
-    def get_K_left(self) -> npt.NDArray[np.float32]:
-        """Returns the intrinsic matrix of the left camera."""
-        left_socket = get_socket_from_name("left", self.name_to_socket)
-        left_K = np.array(
+    def get_K(self, cam_name: str = "left") -> npt.NDArray[np.float32]:
+        """Returns the intrinsic matrix of the requested camera."""
+        assert cam_name in list(
```

Comment:
raising an error would be nicer for the user than an assert.
The same test is done in get_D() why not refactoring this?

Reply:
Done